data/json/items/book/electronics.json

```diff
@@ -168,6 +168,26 @@
     "time": "20 m",
     "fun": -1
   },
+  {
+    "id": "textbook_atomic",
+    "type": "BOOK",
+    "name": "Nuclear Physics Made Easy",
+    "name_plural": "Nuclear Physics Made Easy",
+    "description": "A book detailing the workings of state of the art atomic technology, and the physics behind it.  You could probably figure out how to build a fission reactor from reading it, but after the apocalypse, acquiring suitable machinery and materials for that seems extremely unlikely at best.  However, more fundamental information presented in this book could indeed be useful.",
```

Comment:
Description needs changing, you definitely can not learn to build a working reactor from reading a textbook.